tests/datatypes/test_bytes.py

```diff
@@ -20,6 +20,20 @@ def test_getattr(self):
                 print(err)
             """)
 
+    def test_capitalize(self):
+        self.assertCodeExecution("""
+            print(b'hello, world'.capitalize())
+            print(b'helloWORLD'.capitalize())
+            print(b'HELLO WORLD'.capitalize())
+            print(b'2015638687'.capitalize())
```

Comment:
If you want to be really rigorous here, you should add a test for non-ascii bytes. You can use a `\x` sequence to deliberately specify a "high" byte and a "low" byte (might I suggest `\xc8` as a high byte, so you can subtly sign your name into the test suite :smile:)